src/plugins/condition/ConditionManager.js

```diff
@@ -266,8 +248,28 @@ export default class ConditionManager extends EventEmitter {
         });
     }
 
-    broadcastTelemetry(id, datum) {
-        this.emit(`broadcastTelemetry`, Object.assign({}, this.createNormalizedDatum(datum, id), {id: id}));
+    telemetryReceived(id, datum) {
+        const normalizedDatum = this.createNormalizedDatum(datum, id);
+        const timeSystemKey = this.openmct.time.timeSystem().key;
+        let timestamp = {};
+        timestamp[timeSystemKey] = normalizedDatum[timeSystemKey];
+
+        this.conditionClassCollection.forEach(condition => {
+            condition.getResult(normalizedDatum);
```

Comment:
Will file followup issue for this. We don't need to evaluate all conditions, we should short-circuit if possible here. There are probably good reasons for exhaustively evaluating all of them right now (so we're not caching stale data) but we should figure out how to optimize this.